### DISCUSSION TITLE
Anyone able to build on Windows 11?

### DISCUSSION THREAD

**Discussion by Cr0vis** [immich-app/immich#63] [Q&A]
Just wondering if I’m the only one who’s having problems compiling on windows 11.

**Comment by alextran1502** [comment]
Are you still having problems with WSL on Windows?

**Reply by Cr0vis** [reply]
I guess, I have wsl downloaded and stuff but Immich just won’t compile all the way

**Reply by alextran1502** [reply]
I just found this I think it will help.

[How to enable/disable buildkit in docker?](https://stackoverflow.com/questions/66839443/how-to-enable-disable-buildkit-in-docker)

Seems like you have to disable builtkit in Docker on Windows to make it work

**Reply by Cr0vis** [reply]
still got this
![image](https://user-images.githubusercontent.com/99766020/159137324-121d2a07-d3c2-45fa-af55-144e52672673.png)

**Reply by alextran1502** [reply]
Sorry man, I don't have Windows 11 nor Docker on Windows to test and find the solution for this. Can you post this in the Issue section of the repo, chances that some people run into the same problem and have found a solution will reach out and help
